cfgov/v1/atomic_elements/organisms.py

```diff
@@ -100,6 +100,9 @@ class Meta:
         icon = 'mail'
         template = '_includes/organisms/email-signup.html'
 
+    class Media:
+        js = ["email-signup.js"]
```

Comment:
Should this be `EmailSignup.js` or is there another file with nearly the same name? Also, should it be wrapped in single quotes to match everything else?